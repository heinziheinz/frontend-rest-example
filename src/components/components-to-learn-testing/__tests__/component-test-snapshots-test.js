
import user from './../check-object';
it(
    'check-snapshot', () => {
        // const user = {
        //     createdAt: new Date(),
        //     id: Math.floor(Math.random() * 20),
        //     name: 'Jochen Knochen'
        // };
        expect(user).toMatchSnapshot({
            createdAt: expect.any(Date),
            id: expect.any(Number),
            name: 'Jochen Knochen'
        });
    }
);