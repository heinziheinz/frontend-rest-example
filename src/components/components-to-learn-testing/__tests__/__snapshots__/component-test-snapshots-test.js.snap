// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check-snapshot 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "Jochen Knochen",
}
`;
